packages/web-runtime/src/components/ApplicationsMenu.vue

```diff
@@ -22,7 +22,13 @@
     >
       <ul class="uk-grid-small uk-text-center" uk-grid>
         <li v-for="(n, nid) in menuItems" :key="`apps-menu-${nid}`" class="uk-width-1-3">
-          <a v-if="n.url" key="apps-menu-external-link" :target="n.target" :href="n.url">
+          <a
+            v-if="n.url"
+            key="apps-menu-external-link"
+            :target="n.target"
+            :href="n.url"
+            @click="clickApp(n)"
```

Comment:
Not sure if having a click-handler on an `<a>` tag is optimal here. You could go for the OcButton component and define the type there maybe?

Reply:
Good point! However, this links to an external page, so I'm not sure if we can use a button here (in regards to a11y).

Buttons *should* be used when triggering some kind of backend logic, which is definitely true here. At the same time, buttons *shouldn't* be used when linking to external pages, which is also true here.

Or do you mean using `type="a"` for the button? But it would be rendered as an `a` element anyway, right?

Reply:
I think we'll have to render a second type of list items here, depending on whether they're supposed to be routing (`<a>`)  or trigger an async action (`<button>`)

```
<li v-for="item of menuItems">
  <a v-if="item.target">...</a>
  <button v-else @click="item.clickHandler()">...</a>
</lI>
```

or go for a more sophisticated solution, but this will most likely get a bit messy with (optional) attributes and click handlers

```
<li v-for="item of menuItems">
  <oc-button :type="menuItemType(item)">...</oc-button>
</lI>

...
  computed: {
    return item.target ? 'a' : ''
  }
```

What do you think?

Reply:
What's the reason for adding the click handler to the link anyway? I'd expect a dedicated item in the user menu (not in the app switcher) for going to the default settings in oc10. Not an implicit magical behaviour when going back to the classic UI. 😉

Reply:
Difficulty with that is, that we don't know if web runs from within an oc10 app at all. So we don't exactly know when to show that menu item...

Reply:
@kulmann 
1. Set the default app to oc10
2. forward to oc10
You can see the handler as an interceptor

Reply:
@pascalwengerter 
>I think we'll have to render a second type of list items here, depending on whether they're supposed to be routing (<a>) or trigger an async action (<button>)

It is still doing both, I don't think we need to change something.